### PR TITLE
Create flow test resource example

### PR DESCRIPTION
**Pull request by jdziurzynski**
KNTK-365

Created flow Test resource example and corrected other test examples to work correctly.

### CODE REVIEW

**Review by danielfurman — DISMISSED** [review]

File: docs/resources/test.md [hunk 1]
```diff
@@ -115,10 +115,12 @@ resource "kentik-synthetics_test" "minimal-hostname-test" {
     ]
     ping {
       period = 60
+      count = 1
     }
     trace {
       period   = 60
       protocol = "udp"
```
Comment (mmac-m3a) [inline docs/resources/test.md:122]: With `protocol = "udp"`, the `trace` config must define also `port`. I suggest to change the example and use `protocol = "icmp"`. Applies to all instances.

File: docs/resources/test.md [hunk 2]
```diff
@@ -213,11 +219,13 @@ resource "kentik-synthetics_test" "minimal-dns-test" {
     ]
```
Comment (mmac-m3a) [inline docs/resources/test.md:219]: `dns` and `dns-grid` tests **must** have **only** `dns` in the `tasks` list. Those tests do not support running `ping` and `traceroute`.

File: docs/resources/test.md [hunk 3]
```diff
@@ -251,10 +259,12 @@ resource "kentik-synthetics_test" "minimal-url-test" {
...
```
Comment (mmac-m3a) [inline docs/resources/test.md:259]: `url` test must have `http` task in the task list.

**Review by mmac-m3a — CHANGES_REQUESTED** [review]
The example `flow` test configuration is (mostly) OK. However, I've noticed couple problems in examples of other test types that I've unfortunately missed in previous reviews.

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**